packages/vuetify/src/components/VNavigationDrawer/_variables.scss

```diff
@@ -1,4 +1,27 @@
-@import '../../styles/styles.sass';
+// Defaults
+$navigation-drawer-absolute-z-index: 1 !default;
+$navigation-drawer-background: rgb(var(--v-theme-surface)) !default;
+$navigation-drawer-border-color: $border-color-root !default;
+$navigation-drawer-border-radius: map-get($rounded, '0') !default;
+$navigation-drawer-border-style: $border-style-root !default;
+$navigation-drawer-border-thin-width: thin !default;
+$navigation-drawer-border-width: 0 !default;
+$navigation-drawer-elevation: 0 !default;
+$navigation-drawer-fixed-z-index: 6 !default;
+$navigation-drawer-floating-border: none !default;
+$navigation-drawer-img-height: inherit !default;
+$navigation-drawer-img-object-fit: cover !default;
+$navigation-drawer-img-width: inherit !default;
+$navigation-drawer-overflow-scrolling: touch !default;
+$navigation-drawer-positions: absolute fixed !default;
+$navigation-drawer-temporary-elevation: 16 !default;
+$navigation-drawer-transition-duration: 0.2s !default;
```

Comment:
This and the timing function should be synced with v-main.

Reply:
As far as I can see, it is? 
![image](https://user-images.githubusercontent.com/9064066/110723900-70c8e600-81da-11eb-8aca-97d3b75625ed.png)